install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -109,6 +109,122 @@ key "update-key" {
 
 NOTE: The key name may vary and the above is only an example.
 
+
+To enable Kuryr SDN instead of the default openshift-sdn, the following all.yaml
```

Comment:
Suggestion for the paragraph:

The following [filename]`all.yml` file  enables Kuryr SDN instead of the default openshift-sdn.
Note that  the example below  is a condensed version, so it is
important to review the default template carefully.